deploy/010-deploy-reward-distributors.ts

```diff
@@ -56,6 +56,11 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
         log: true,
         skipIfAlreadyDeployed: true,
       });
+
+      const rewardsDistributor = await ethers.getContract(contractName);
+      if (rewardsDistributor.address !== proxyOwnerAddress && rewardsDistributor.owner() === deployer) {
```

Comment:
I don't understand the first check. When the address of the RewardsDistributor could be equal to the `proxyOwnerAddress`?